packages/docs/docs/concepts/progressive-loading/volumeProgressive.md

```diff
@@ -0,0 +1,134 @@
+# Create JLS and JLS thumbnail versions
+mkdicomweb create -t jhc --recompress true --alternate jls --alternate-name jls /src/viewer-testdata/dcm/Juno
+mkdicomweb create -t jhc --recompress true --alternate jls --alternate-name jlsThumbnail --alternate-thumbnail /src/viewer-testdata/dcm/Juno
+# Create HTJ2K lossless and thumbnail versions (this is not required in general
+# when the top item is already lossless)
+mkdicomweb create -t jhc --recompress true --alternate jhcLossless --alternate-name htj2k  /src/viewer-testdata/dcm/Juno
+mkdicomweb create -t jhc --recompress true --alternate jhc --alternate-name htj2kThumbnail --alternate-thumbnail /src/viewer-testdata/dcm/Juno
+```
+
+Any other tools creating multipart/related encapsulated data can be used, as
+can using accept headers or parameters for a standard DICOMweb server.
+
+Note the data path for these is, in general the normal DICOMweb path with
+`/frames/` replaced by some other name.
+
+# Configuration
+
+See the volumeProgressive example for full details.
+
+No special configuration is required for HTJ2K images streaming, but that isn't
+particularly effective. The byte range request is preferably, but requires
+additional server side support for byte ranges. Other types require custom
+configuration.
+
+## HTJ2K Byte Range Configuration
+
+```javascript
+cornerstoneDicomImageLoader.configure({
+  retrieveOptions: {
+    default: {
+      '3.2.840.10008.1.2.4.96': {
+        streamingDecode: true,
+      },
+      default: {},
+    },
+    multipleFinal: {
+      default: {
+        range: 1,
+      },
+    },
+    multipleFast: {
+      default: {
+        // Note it is the decode that is streaming, but a range request to
+        // get the data, not a streaming request.
+        streamingDecode: true,
+        range: 0,
+        initialBytes: 64000,
+        decodeLevel: 0,
+      },
+    },
+  },
+```
+
+# Performance
+
+The performance gains on this vary quite a bit depending on size of data
+and capabilities of the DICOMweb server components. In general, a 350% speed improvement
+to first volume render is seen because of the interleaved images filling
+nearby images. For straight JLS versus HTJ2K, that depends on the compression
+size - JLS compresses a bit better than HTJ2K, resulting in faster downloads.
+However, HTJ2K is faster decoding, resulting in lower times from download
+until image rendering.
+
+For reduced resolution images, both the HTJ2K and JLS versions are slower
+to final image data than the original versions because there are additional
+requests required to fetch all data. HTJ2K decoding still has some decompression
+issues on very small thumbnails resulting in some issues decoding small
+reduced resolution versions. That results in higher time to first volume times.
+
+Note that none of the times include time to load the decoder, which can be
+a second or more, but is only seen on first render. These times are similar for
+both types.
```

Comment:
We can pre-load the WASM files via a configuration in the `.initialize()` of the `CornerstoneDICOMImageLoader`. Maybe we should mention that.